### ISSUE TITLE
VS Code: After installing C# extension and creating blazor project will meet warning: "Found markup element with unexpected name 'PageTitle'".

### ISSUE THREAD

**Issue by v-wenkecheng** [OmniSharp/omnisharp-vscode#5630]
VS Code: 1.76.1
C# Extension version:[1.25.5](https://github.com/OmniSharp/omnisharp-vscode/releases)
SDK: [8.0.0-Preview.3.23164.2](https://github.com/dotnet/installer)

**Platform** 
- [x] Windows
- [x] Linux
- [x] Mac 

**To reproduce:** 
1. Open a normal Terminal Emulator, create MVC project and open it using VS code
```
md Blazor
cd Blazor
dotnet new blazorserver
code .
```
2. Go to [https://github.com/OmniSharp/omnisharp-vscode/releases](https://github.com/OmniSharp/omnisharp-vscode/releases) > Download the csharp-1.25.5-win32-x64.vsix.
3. In VS Code and go to the Extensions Tab, select "Install from VSIX" > install the file you downloaded.
4. Navigate to Explorer (at the upper left corner) > Open Pages/index.razor, verify there is no any error in PROBLEMS window.

**Actual behavior:**
"Found markup element with unexpected name 'PageTitle'. If this is intended to be a component, add a @using directive for its namespace.".

![image](https://user-images.githubusercontent.com/69494255/225247018-ec23e552-ce91-4ec1-b05a-1ca876ae0a11.png)

**Expected behavior:**
![image](https://user-images.githubusercontent.com/69494255/225247201-3513a80a-8be7-4a22-927e-936117d9069e.png)

**Additional context:**
Note: 
1. This issue does not repro on C# extension 1.25.4.
2. This issue only repro on C# extension 1.25.5.

**Comment by XorZy** [comment]
I am also seeing this on Arch with .NET 7.0.102.

**Comment by TrieBr** [comment]
This seemed to have fixed after upgrading to extension 1.25.8 for me. At least for now..

**Comment by JoeRobich** [comment]
This issue seems to be resolved when using .NET 8 Preview 4. Please reopen issue if it pop back up.

**Comment by Neogamezzz** [comment]
Same problem

**Comment by lperry65** [comment]
I'm using VS Code on Mac, I've made sure C# extensions are up to date. I've deleted bin and obj folders and done a clean and a build. Nothing works, still get the warnings.

Found markup element with unexpected name 'PageTitle'. If this is intended to be a component, add a @using directive for its namespace.Razor(RZ10012)
<img width="1197" alt="Screenshot 2024-07-23 at 10 23 13" src="https://github.com/user-attachments/assets/e25774a9-460c-4e5f-ad7e-20534d689c3a">
